jekyll/_cci2/container-runner.adoc

```diff
@@ -95,6 +95,184 @@ agent:
           - name: "other"
 ```
 
+[#customizable-service-containers]
+=== Customizable service containers
+
+By default, link:https://circleci.com/docs/using-docker/#using-multiple-docker-images[service (or secondary) containers] inherit the same link:https://kubernetes.io/docs/tasks/configure-pod-container/[container configuration] as defined by the primary container. However, this behavior can be overridden using customizable service containers. This allows fine-tuned control over a service's resource usage on a per-image basis.
+
+[#customizable-service-containers-example]
+==== Example
+
+Consider the following container runner Helm values:
+
+[source,yaml]
+----
+agent:
+  serviceContainers:
+    exact:
+      - "cimg/redis:6":
+          spec:
+            resources:
+              requests:
+                cpu: "0.5"
+                memory: "200Mi"
+  resourceClasses:
+    your-namespace/your-resource-class:
+      serviceContainers:
+        exact:
+          - "cimg/postgres:16":
+            spec:
```

Comment:
This gets mapped straight onto a container struct, so we don't need `spec:`

Reply:
This is what happens when you get ChatGPT to generate an example for you.